### DISCUSSION TITLE
RAD REU PCB Pick and Place file?

### DISCUSSION THREAD

**Discussion by deanflyer** [frntc/RAD#43] [Q&A]
I've ordered a RAD REU PCB from PCBWay, and I am using their assembly service. They are asking for a pick and place (centroid) file? Is this something they need to proceed with the order. Sorry for the stupid question!

**Comment by frntc** [comment]
There was essentially the same thing here: https://github.com/frntc/RAD/discussions/38

The problem is that I don't have a PCBWay-compatible centroid file and to make sure I have a working one, I'd need to order a >=1 test batch(es) myself. It is a bit annoying that PCBWay is trying to sell pre-assembled PCBs without having the data to do so... (I have no control over that!)

Maybe they have a centroid file for one of the PCBs (placement is the same for both), or they are willing to create if themselves from https://htmlpreview.github.io/?https://github.com/frntc/RAD/blob/master/Gerber/ibom.html

**Reply by deanflyer** [reply]
Thanks for the quick response. I've reached out to PCBWay, let's see what they say. Hopefully they can complete the order even if it costs extra.

The annoying thing is when you submit the order with the BOM it came back as everything is fine, its only after you pay for it that they have requested a centroid file.

**Reply by deanflyer** [reply]
PCBWay got back to me and said the request for a pick and place/centroid file was a system generated message and I should ignore it. The order will be completed and the SMDs will be placed manually. Hopefully this is of use to anyone else looking to get PCBWay to do the assembly.

**Reply by 8-BitBoyz** [reply]
@deanflyer did your build come out ok and working in the end? 

I just got the message also just now from pcbway looking for that same file.
